package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hmt-escrow",
-  "version": "0.11.0",
+  "version": "0.12.0",
```

Comment:
You've changed a patch version, while we have a breaking change here. We need to change a major version. https://semver.org/

Reply:
Do you mean `1.0.0` version?

Reply:
yep